services/user.py

```diff
@@ -0,0 +1,46 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()

```

Comment:
You're calling `save()` method multiple times. It's not efficient and it's against our checklist. You should call `save()` only once after all the changes. In your case, you can call `save()` method right before returning the user object.